src/components/common/names-capture.vue

```diff
@@ -874,26 +875,28 @@ export default class NamesCapture extends Vue {
   }
 
   @Watch('request_action_cd')
-  updateLocationOnAssumedName (newVal, oldVal) {
-    if (newVal === 'ASSUMED') {
-      if (this.location === 'BC') this.location = 'CA'
+  updateLocationOnAssumedName (newVal: RequestCode, oldVal: RequestCode) {
+    if (newVal === RequestCode.ASSUMED && this.location === Location.BC) {
+      this.location = Location.CA
     }
   }
 
   @Watch('entity_type_cd')
-  handleEntityType (newVal, oldVal) {
-    if (newVal === 'INFO') {
+  handleEntityType (newVal: EntityType, oldVal: EntityType) {
+    // special case for sub-menu
+    if (newVal === EntityType.INFO) {
       this.setPickEntityModalVisible(true)
       this.entity_type_cd = oldVal
     }
   }
 
   @Watch('location')
-  handleLocation (newVal, oldVal) {
+  handleLocation (newVal: Location, oldVal: Location) {
     if (newVal !== oldVal) {
       this.setNRData({ key: 'xproJurisdiction', value: '' })
     }
-    if (newVal === 'INFO') {
+    // special case for sub-menu
+    if (newVal === Location.INFO) {
```

Comment:
I'm not 100% sure, but i don't believe Location has an INFO value anymore.

No change required, but this might be some left over from previous iterations where there was a helper choice in the Location drop down. Just thinking out loud :)

Reply:
You're probably right, but I don't want to change that atm :)